### ISSUE TITLE
[Blog-1][Review]- Is Python right for you?

### ISSUE THREAD

**Issue by SwastikaYadav** [tanaypratap/teamtanay.jobchallenge.dev#681]
Hello Mentors
I hope you are doing well.
This is my first blog -"Is Python right for you?"

Kindly review it and let me know if there is any suggestions.
Blog link:  [https://dev.to/swastikayadav/is-python-right-for-you-pp2](https://dev.to/swastikayadav/is-python-right-for-you-pp2)

**Comment by SwastikaYadav** [comment]
Please review the blog and let me know the suggestions.

**Comment by gkanishk** [comment]
Hey @SwastikaYadav ,
Just read your blog nicely written Kudos to you, a few things I want to tell:  
- It's good that you have mentioned why not to go for python but if you add some latest stats from the surveys like the `Stackoverflow 2020` developer survey it would be great.
- Also While you have mentioned some characteristics for which we should not go for python there try to relate that with other stacks used like mobile development we could go for react-native as that's fast and integration is better, similarly mention for other points  
- In short, connect and compare that with other stacks that a better alternative is available and try to provide statistics for better reasonability.
